src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -451,8 +453,13 @@ function MoneyRequestReportPreviewContent({
         if (isPaidAnimationRunning) {
             return CONST.REPORT.REPORT_PREVIEW_ACTIONS.PAY;
         }
+
+        if (isSubmittingAnimationRunning) {
+            return CONST.REPORT.REPORT_PREVIEW_ACTIONS.SUBMIT;
+        }
+
```

Comment:
Same here, we should move this and the `isPaidAnimationRunning` logic inside `getReportPreviewAction`